### ISSUE TITLE
How to change test event? 

### ISSUE THREAD

**Issue by MikeUdin** [revmischa/sublime-awslambda#10]
Hi! Thank you for the nice plugin!
Is it possible to change test event parameter?

For example:  
`def lambda_handler(event, context):
return event`

returns in console Result: {"sublime": true}
would be great to setup custom json events to use with 'Invoke Function'

**Comment by revmischa** [comment]
I agree setting up custom test data (like in the web UI) would be super handy. Feel free to send a PR!
